src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -104,7 +104,9 @@ def create(self, size, isolate_nspace, pool, mode, uid, gid, earmark):
                 'data_pool': pool,
                 'pool_namespace': self.namespace if isolate_nspace else None,
                 'quota': size,
-                'earmark': earmark
+                'earmark': earmark,
+                'case_sensitive': case_sensitive,
+                'normalization': normalization
```

Comment:
```suggestion
                'normalization': normalization,
```

be the change you want to see in the world

Reply:
I don't understand this change. Why do you want to add a `,` after the last item. Previously it wasn't there.

Reply:
because this diff touches lines not directly related to this change only to add a comma. If you add a comma here for the last item, future additions won't have to touch this "normalization" line (and thereby pollute the `git blame` output).

Reply:
Ok. I applied the same approach to some other definitions as well.